### PR TITLE
Log error in Admin for no servers

### PR DESCRIPTION
**Pull request by milleruntime**
* Also improve stop all description

### CODE REVIEW

File: server/base/src/main/java/org/apache/accumulo/server/util/Admin.java [hunk 1]
```diff
@@ -115,7 +115,7 @@ static class StopManagerCommand {}
   @Parameters(commandDescription = "stop the master (DEPRECATED -- use stopManager instead)")
   static class StopMasterCommand {}
 
-  @Parameters(commandDescription = "stop all the servers")
+  @Parameters(commandDescription = "stop all of the tablets servers and manager")
```
Comment (ctubbsii) [inline server/base/src/main/java/org/apache/accumulo/server/util/Admin.java:118]:
```suggestion
  @Parameters(commandDescription = "stop all tablet servers and the manager")
```

Comment (ctubbsii) [reply server/base/src/main/java/org/apache/accumulo/server/util/Admin.java:118]: @milleruntime Do you know if this stops the accumulo-gc and accumulo-monitor, or any of the compactor services? I would expect it to stop everything, including those, but based on the description you changed it to, I get the impression it doesn't.

If it only stops the manager and tservers, I would like to commit my additional wording changes, but if it does stop those other servers, then the change you made should be rolled back instead.

Comment (milleruntime) [reply server/base/src/main/java/org/apache/accumulo/server/util/Admin.java:118]: It was not stopping the GC or monitor. I don't know about the new services.

Comment (ctubbsii) [reply server/base/src/main/java/org/apache/accumulo/server/util/Admin.java:118]: Okay, then fixed in c4b0ee33dee4863c34a8532c85bef9852bff5895

**Pull request merged**